src/main/java/org/spongepowered/plugin/meta/SpongeExtension.java

```diff
@@ -0,0 +1,15 @@
+package org.spongepowered.plugin.meta;
+
+public final class SpongeExtension {
+
+    public String assets; // public for mixin compatibility
+
+    public SpongeExtension(String assets) {
```

Comment:
`PluginMetadata` is using null so unnecessary entries are not created in the JSON output (keep in mind Forge never serializes, it only deserializes). The no-arg constructor is just for consistence, don't want to add 5 new constructors to `PluginMetadata`